waspc/src/Wasp/Generator/Test.hs

```diff
@@ -7,16 +7,18 @@ import Control.Concurrent (newChan)
 import Control.Concurrent.Async (concurrently)
 import StrongPath (Abs, Dir, Path')
 import System.Exit (ExitCode (..))
-import Wasp.Generator.Common (ProjectRootDir)
 import Wasp.Generator.Job.IO (readJobMessagesAndPrintThemPrefixed)
 import qualified Wasp.Generator.WebAppGenerator.Test as WebAppTest
+import Wasp.Project.Common (WaspProjectDir)
 
-testWebApp :: [String] -> Path' Abs (Dir ProjectRootDir) -> IO (Either String ())
+testWebApp :: [String] -> Path' Abs (Dir WaspProjectDir) -> IO (Either String ())
 testWebApp args projectDir = do
   chan <- newChan
   let testWebAppJob = WebAppTest.testWebApp args projectDir chan
   (testExitCode, _) <-
     testWebAppJob `concurrently` readJobMessagesAndPrintThemPrefixed chan
   case testExitCode of
     ExitSuccess -> return $ Right ()
+    -- Exit code 130 is thrown when user presses Ctrl+C.
+    ExitFailure 130 -> return $ Right ()
```

Comment:
I'm not sure, but when you CTRL + C to stop the watching of tests, it outputs an ugly message "It failed with code 130" so I wanted to avoid showing that error to the user.

Reply:
I've updated `vitest` so maybe their behaviour changed and now we had to handle this case.